README.md

```diff
@@ -37,7 +37,6 @@ mvn clean install -Pscala-2.13
 ./mvn clean verify -Pcode-coverage
 ```
 Code coverage will be generated on path:
-```aidl
```

Comment:
Why?
The backticks are now imbalanced after this change....

Reply:
It was unsuccessful try to remove wrong tag using suggestion feature. It failed. Problem has been solved in commit 9cdca671a9d28249f6d958bdf41df997d8f6b1ee.